charts/streams-app/README.md

```diff
@@ -63,6 +63,7 @@ Alternatively, a YAML file that specifies the values for the parameters can be p
 | `kafka.outputTopic`        | Output topic for your streams application.                                                                                                                                       |         |
 | `kafka.extraOutputTopics`  | Map of additional named output topics if you need to specify multiple topics with different message types.                                                                       | `{}`    |
 | `kafka.errorTopic`         | Error topic for your streams application.                                                                                                                                        |         |
+| `kafka.applicationId`      | Unique application ID for Kafka Streams. Required for auto-scaling                                                                                                               |         |
```

Comment:
should we name it applicationId because streams calls it like this or better consumerGroup in order to be more generic, maybe we add a pure consumerapp in the future?

Reply:
Good point, but then it should probably be called groupId: https://kafka.apache.org/documentation/#consumerconfigs_group.id

Reply:
I renamed it but am still not entirely convinced. wdyt?

Reply:
Ok, I decided to use application id. It makes the most sense in context of Kafka Streams and we don't have any pure consumer apps right now